### PR TITLE
Updating readme.md to run example

### PR DESCRIPTION
**Pull request by jaskiratr**
As discussed in #139, added a few steps in `readme.md` to install dependencies prior to running the example in the repository. 
Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,7 +145,7 @@ you should generate documentation with all renderers:
 
 ```bash
 $ cd example
-$ yarn
+$ yarn install
 $ yarn docs
```
Comment (Kocal) [inline README.md]: What about removing every `yarn docs:...` for only one `yarn docs:all`?

File: README.md [hunk 2]
```diff
@@ -119,6 +119,19 @@ Feel free to open an issue/pull request if your template is not supported!
...
+
+```bash
+$ git clone https://github.com/Kocal/jsdoc-vuejs
+$ cd jsdoc-vuejs
+$ yarn install
+
+# For testing the example docs
+$ cd example
+$ yarn install
+$ yarn docs
```
Comment (Kocal) [inline README.md]:
```suggestion
$ yarn docs # output in `docs` folder
```

File: README.md (hunk 1 above)
Comment (Kocal) [reply README.md]: I understand, but I don't think this is useful here in E2E context.

When you update the source code, you want to generate example docs for every renderers (default, docstrap, ...), not only one. 
To be honest, I stopped using `yarn docs:docstrap` or `yarn docs:tui` when `yarn docs:all` was introduced by @silverspectro.

Maybe we can change to this:

```bash
$ cd example
$ yarn install

# generate docs for every renderers
$ yarn docs:all

# or one by one
$ yarn docs
$ yarn docs:docstrap
$ yarn docs:minami
$ yarn docs:tui
```

Comment (jaskiratr) [reply README.md]: Affirmative. You make a good point. The suggestion is clear as well.

**Comment by Kocal** [comment]
Hi @jaskiratr, do you have some time to finish this pull request? 
If no I can take over! :slightly_smiling_face:

**Comment by jaskiratr** [comment]
Hi @Kocal 
Updated the `Readme.md` per your last suggestion.

**Comment by Kocal** [comment]
Thanks!

**Pull request merged**